src/app/destinyTrackerApi/reviewsFetcher.ts

```diff
@@ -1,12 +1,72 @@
 import { ItemTransformer } from './itemTransformer';
 import { PerkRater } from './perkRater';
 import { UserFilter } from './userFilter';
-import { D1ItemReviewResponse, D1CachedItem } from '../item-review/destiny-tracker.service';
 import { ReviewDataCache } from './reviewDataCache';
 import { handleErrors } from './trackerErrorHandler';
 import { loadingTracker } from '../ngimport-more';
 import { D1Item } from '../inventory/item-types';
 import { dtrFetch } from './dtr-service-helper';
+import { D1ItemReviewResponse, D1ItemUserReview } from '../item-review/d1-dtr-api-types';
+import { DtrReviewer } from '../item-review/dtr-api-types';
+
+/** A single user's review for a D1 weapon. */
+export interface ActualD1ItemUserReview {
```

Comment:
Why is this an "actual" item user review?

Reply:
It's the name I gave to distinguish between what we actually receive from the D1 API and what we cache as having received internally.

Reply:
Which one is this?